crates/task-impls/src/events.rs

```diff
@@ -43,12 +43,15 @@ pub enum HotShotEvent<TYPES: NodeType> {
     /// A Data Availability Certificate (DAC) has been recieved by the network; handled by the consensus task
     DACRecv(DACertificate<TYPES>),
     /// A DAC is validated.
-    DACValidated(DACertificate<TYPES>),
+    DACertificateValidated(DACertificate<TYPES>),
```

Comment:
+1 on the name change

should we also change `DACRecv` to `DACertificateRec` too for consistency? (this is also very minor so we can just do it later)